.env

```diff
@@ -0,0 +1,4 @@
+NAME=postgres
```

Comment:
It's better to add `.env` file to `.gitignore` list.
Any `env` files should be only your local files.